### PR TITLE
Updated ingame guide

### PR DESCRIPTION
**Pull request by Peptide90**
Includes more indepth gameplay description and definitions for antag and IC / OOC.

Good for new players to SS13 / SS14 or roleplay in general. Formatting is poo though as couldn't get headings as per the[ design doc here](https://docs.google.com/document/d/1Qi3GlBwjY27IOLRNzme5xUCVskJd_bu3DpGSS8WOs1U/edit?usp=sharing). Also did not include the job sections or antag guide at the risk of flooding the window with even more text that nobody will read.

Forcing people to read this for X amount of time as a punishment would be a nice admin tool as well as updating the window to just be able to support a HTML document or web page so we can insert pictures and formatting easier like a proper guide.

Please feel free to edit what I've included or just knock this back entirely until we have a better system in place. I'm only thinking of the large playtest coming up tomorrow.

:cl:
- add: More in depth tutorial. Press F1 to learn some basics about the server. More to come in future.

### CODE REVIEW

**Review by MrOrangeMan — APPROVED** [review]
Man, I needed some of those explanations so hard xD

File: Resources/Server Info/Intro.txt [hunk 1]
```diff
@@ -1 +1,8 @@
...
\ No newline at end of file
+Welcome to space station 14! You have just started your shift aboard a Nanotrasen space station and along with your crewmates should cooperate, perform your job role to the best of your ability and have fun (not at the expense of others hopefully).
+
+Please make sure you understand the server rules (F1) and if you have any questions, consult this guide, your colleagues on shift in person or over radio or via discord. You can also ask admins for help (ahelp) by hitting the escape key > admin help and typing a message to any online admins. Especially do this if you witness someone breaking rules.
+
+To reiterate, you are all a crew of people who work together (even the clown) and you should not be harming each other unless you have an extremely good reason (self defence) and even then, call security over radio, stun and detain, do not commit murder as a non antagonist (see antagonist further down).
+
+
+Instead of intents, we have combat mode. Check controls above for its keybind. You can't attack anybody with it off, so no more hitting yourself with your own crowbar.
```
Comment (moonheart08) [inline Resources/Server Info/Intro.txt:8]: Should say controls BELOW

**Review by moonheart08 — APPROVED** [review]

File: Resources/Server Info/Intro.txt (hunk 1 above)
Comment (Peptide90) [reply Resources/Server Info/Intro.txt:8]: Thanks, copied that bit from the old guide

Comment (MrOrangeMan) [reply Resources/Server Info/Intro.txt]: Maybe add something like "Or look at the slots on the left" (My English grammar is terrible, but you get the idea)

**Pull request merged**